src/node_process_object.cc

```diff
@@ -105,18 +105,37 @@ MaybeLocal<Object> CreateProcessObject(Realm* realm) {
   READONLY_PROPERTY(
       process, "version", FIXED_ONE_BYTE_STRING(isolate, NODE_VERSION));
 
-  // process.versions
   Local<Object> versions = Object::New(isolate);
-  READONLY_PROPERTY(process, "versions", versions);
+  // Node.js version is always on the top
+  READONLY_STRING_PROPERTY(
+      versions, "node", per_process::metadata.versions.node);
+  std::vector<std::pair<std::string_view, std::string_view>> versions_vec;
```

Comment:
The number of keys is known upfront so it would be better to use a stack-allocated array instead of a std::vector because the latter always allocates heap memory. Example:
```c++
#define V(key) +1
std::pair<std::string_view, std::string_view> versions_array[NODE_VERSIONS_KEYS(V)];
#undef V

auto* slot = &versions_array[0];
#define V(key) *slot++ = std::make_pair(#key, per_process::metadata.versions.key);
#undef V

std::sort(&versions_array[0],
          &versions_array[arraysize(versions_array)],
          [](auto& a, auto& b) { /* ... */ });
```